keyvi/src/cpp/dictionary/dictionary.h

```diff
@@ -88,9 +88,9 @@ final {
      * @param key The key
      * @return True if key is in the dictionary, False otherwise.
      */
-    bool Contains(const char* key) const {
+    bool Contains(const std::string& key) const {
       uint64_t state = fsa_->GetStartState();
-      size_t key_length = strlen(key);
+      size_t key_length = key.size();
```

Comment:
In general I prefer and have a habit to define variables which are not going to be changed as `const`. This improves readability, prevents from some errors and even sometimes can help compiler to generate more efficient code.